src/selenium/schedulePage.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const SchedulePage = Object.create(BasePage);
+const datesId = ['2017-03-17', '2017-03-18', '2017-03-19'];
+
+SchedulePage.checkIsolatedBookmark = function() {
+  const self = this;
+  const bookmarkSessionsIdsArr = ['3015'];
+  const visibleCheckSessionsIdsArr = ['3014', '3015', '2918'];
+
+  return self.bookmarkCheck(bookmarkSessionsIdsArr, visibleCheckSessionsIdsArr);
+};
+
+SchedulePage.toggleSessionElem = function() {
+  const self = this;
+  // Checking the toggle behaviour of session having id 3014
+  const promise = new Promise(function(resolve) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)